etl/steps/data/meadow/wb/2024-06-26/poverty_projections.py

```diff
@@ -0,0 +1,57 @@
+"""Load a snapshot and create a meadow dataset."""
+
+import owid.catalog.processing as pr
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshots.
+    snap_number = paths.load_snapshot("poverty_projections_number_global.csv")
+    snap_share = paths.load_snapshot("poverty_projections_share_regions.csv")
+
+    # Load data from snapshot.
+    tb_number = snap_number.read()
+    tb_share = snap_share.read()
+
+    #
+    # Process data.
+    #
+    # Rename columns
+    tb_number = tb_number.rename(columns={"million_poor": "headcount_215"})
```

Comment:
Nitpicky: I would either multiply with a million here or rename this column only when you multiply